game/src/lib.rs

```diff
@@ -155,6 +155,9 @@ fn setup_app(
     {
         opts.color_scheme = map_gui::colors::ColorSchemeChoice::NightMode;
     }
+    if title {
+        opts.color_scheme = map_gui::colors::ColorSchemeChoice::Pregame;
```

Comment:
I think this breaks the ability to override the color scheme from the command line - is that OK? (It's fine with me)

Reply:
Yes, but it was already pretty broken in `game`. Even if you skip the title screen with `--dev`, some of the startup logic for jumping to 6am fiddles with day/night mode. I can repair this later if it'd be useful, but I have my doubts -- the flag is mostly there for quickly debugging/iterating on a new CS.

I verified `--color_scheme` still works in the other apps, which don't have all of this day/night switching.